apps/src/templates/sectionProgressV2/section-progress-refresh.scss

```diff
@@ -0,0 +1,65 @@
+.legend {
+    align-items: flex-start;
+    border: 1px solid;
+    border-color: var(--lightgray-200);
```

Comment:
You can use `$light_gray_200` if you have `@import "color.scss";` at the top of the file.

nit: I also prefer the style `border: 1px solid $light_gray_200;` instead of defining on two lines